cob_relayboard/ros/src/cob_relayboard_node.cpp

```diff
@@ -93,11 +94,12 @@ class NodeClass
   // Constructor
   NodeClass()
   {
-    n = ros::NodeHandle("~");
+    n = ros::NodeHandle();
+    n_priv = ros::NodeHandle("~");
 
-    topicPub_isEmergencyStop = n.advertise<cob_msgs::EmergencyStopState>("/emergency_stop_state", 1);
-    topicPub_PowerBoardState = n.advertise<cob_msgs::PowerBoardState>("/power_board/state", 1);
-    topicPub_Voltage = n.advertise<std_msgs::Float64>("/power_board/voltage", 10);
+    topicPub_isEmergencyStop = n.advertise<cob_msgs::EmergencyStopState>("emergency_stop_state", 1);
+    topicPub_PowerBoardState = n.advertise<cob_msgs::PowerBoardState>("power_board/state", 1);
+    topicPub_Voltage = n.advertise<std_msgs::Float64>("power_board/voltage", 10);
```

Comment:
we took that as it has been...but it clearly makes sense to harmonize his...done!
(+ tabs vs. spaces)